modules/fancyBox/source/jquery.fancybox.js

```diff
@@ -1865,7 +1867,36 @@
 				return;
 			}
 
-			title = $('<div class="fancybox-title fancybox-title-' + type + '-wrap">' + text + '</div>');
+			if (['inside', 'outside', 'over'].includes(type)) {
+				var parts = [text, link].filter(Boolean);
+				title = $('<div class="fancybox-title fancybox-title-' + type + '-wrap">' + parts.join(sep) + '</div>');
+			}
+			else { // 'float'
+				title = $('<div class="fancybox-title fancybox-title-' + type + '-wrap">' + text + '</div>');
+
+				title.appendTo('body');
+
+				if (IE) {
+					title.width( title.width() );
+				}
+
+				var zeroWidthSpace = '<wbr>'; // To preserve -webkit-box sizing and positioning when text is empty
+
+				title.html(function (index, oldhtml) {
+					return `
+						<span class="child">
+							<div class="container">
+								<span class="text-expanded">${zeroWidthSpace + oldhtml}</span>
+								<span class="link">${(oldhtml ? sep : "") + link}</span>
+								<span class="text-collapsed">${zeroWidthSpace + oldhtml}</span>
+							</div>
+						</span>
+					`;
```

Comment:
Dit werd door FF en Chome zo niet geaccepteerd. Ik weet niet waarom. Heb het vervangen door 

```js
                                        return '<span class="child">' +
                                                        '<div class="container">' +
                                                                '<span class="text-expanded">' + zeroWidthSpace + oldhtml + '</span>' +
                                                                '<span class="link">' + (oldhtml ? sep : "") + link + '</span>' +
                                                                '<span class="text-collapsed">' + zeroWidthSpace + oldhtml + '</span>' +
                                                        '</div>' +
                                                '</span>';

```

en dan werkt het in elk geval. Geen idee wat de "echte" fix is?